src/clip.sol

```diff
@@ -352,9 +358,10 @@ contract Clipper {
 
     // Cancel an auction during ES
     function yank(uint id) external auth {
-        require(sales[id].usr != address(0), "Clipper/usr-not-set");
-        dog.digs(ilk, sales[id].tab);
-        vat.flux(ilk, address(this), msg.sender, sales[id].lot);
+        Sale memory sale = sales[id];
```

Comment:
I came a bit late with this. But it should be less gas efficient than before.
If I'm not wrong doing `Sale memory sale = sales[id];` will do 5 `SLOAD`s when before it was just doing 3.

Reply:
You are right, it is more gas efficient, I'll send another PR to fix it and add a test for `yank` that can be used to measure the gas in the future.